Adding Streams information into the Manage data section (#3399)
elastic/docs-content: manage-data/lifecycle/data-stream/tutorial-data-stream-retention.md

# Tutorial: Configure data stream retention [tutorial-manage-data-stream-retention]

> In this tutorial, we are going to go over the data stream lifecycle retention; we will define it, go over how it can be configured and how it gets applied. Keep in mind, the following options apply only to data streams that are managed by the data stream lifecycle.

Reviewer: ```suggestion
This tutorial demonstrates lifecycle retention, showing how to define, configure, and apply it. Keep in mind that the following options apply only to data streams that are managed by data stream lifecycle.
```

I might rephrase this a bit to avoid the "royal we" and also to avoid the duplication of "go over".

2. Ensure that the lifecycle is enabled, meaning this should be `true`.

:::{tip}
:applies_to: {"stack": "ga 9.2, preview 9.1", "serverless": "ga"}

> You can also review how a data stream is managed by locating it on the **Streams** page in {{kib}}. A stream directly corresponds to a data stream. Select a stream to view its details and go to the **Retention** tab.

Author: I've reread all my previous edits and I went with `Each stream maps directly to an {{es}} data stream.`
